clang/test/CodeGen/loop-alias-mask.c

```diff
@@ -0,0 +1,404 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
```

Comment:
We generally don't add source level tests for llvm optimizations; aiui clang's CodeGen test directory is to test lowering the AST to LLVM IR, not end-to-end.

We might want a different way of testing the feature from source that we could add to a buildbot.